.travis.yml

```diff
@@ -146,21 +146,21 @@ script:
 # libtorrent is the name of the test suite target
   - cd test
   - 'if [ "$tests" == "1" ]; then
-      travis_retry bjam -j3 warnings-as-errors=on ssl=$ssl crypto=$crypto debug-iterators=on picker-debugging=on invariant-checks=full $toolset variant=$variant libtorrent test_natpmp enum_if -l300 &&
+      travis_retry bjam -j3 c++-template-depth=512 warnings-as-errors=on ssl=$ssl crypto=$crypto debug-iterators=on picker-debugging=on invariant-checks=full $toolset variant=$variant libtorrent test_natpmp enum_if -l300 &&
```

Comment:
if you add this to the "~/user-config.jam" file instead, you only have to mention it in 3 places. For ``darwin``, ``gcc`` and ``gcc-sanitizer``. I think it might make this script read a bit easier too (come to think of it, I should probably move some other common features into the bjam config file)